Assignment4/Assignment4_QNo3/SolAssignmentNo4_QNo3.rb

```diff
@@ -0,0 +1,42 @@
+# => 3. Write a program to convert each key value as
+# => separate hash and return an array of hashes.
+# => eg: Input: {:a => 1, :b => 2, :c => 3}
+# => Output: [{:a => 1}, {:b => 2}, {:c => 3}]
+class CustomClass
+  def get_hash_op
+    # => create an empty hash
+    var_hash = Hash[]
+    # => Enter the elements into hash
+    res = "y"
+
+    while (res == "y") do
+      # => input hash key as string
+      puts "Enter the hash key"
+      h_key = gets.chomp
+      # => input hash value as integer
+      puts "Enter the hash value"
+      h_value = gets.chomp.to_i
+      # => inserting into the hash
+      var_hash.store(h_key,h_value)
+      puts "------------------"
+      puts "Enter y/n to continue/exit"
+      res = gets.chomp
+    end
+
+    # => print original hash
+    puts "Before modification of Hash #{var_hash.to_hash}"
+    arr = Array.new(var_hash.to_a)
```

Comment:
@HrisikeshSinghaChowdhury ,
No need to initialise an array in this way`arr = Array.new(var_hash.to_a)`. Initialise array like arr = []
`Array.new(var_hash.to_a)`   .to_a already convert hash to array then why do you want to keep it array once again?

Reply:
ok and done